docs/cardinal/game/message.mdx

```diff
@@ -61,6 +66,42 @@ You can easily create a new message and register it to the world by following th
 </Steps>
 
 --- 
+
+## Message Options
+
+### Custom Message Group
+
+By default, messages are registered under the "game" group, which maps it to the `/tx/game/{txName}` route. You can specify a custom group name using `WithCustomMessageGroup` when you register your messages.
+
+```go
+import (
+    "pkg.world.dev/world-engine/cardinal"
+    "pkg.world.dev/world-engine/cardinal/message"
+)
+
+// This will register your message at `/tx/player/attack-player`
+cardinal.RegisterMessage(w, "attack-player", message.WithCustomMessageGroup[msg.AttackPlayerMsg, msg.AttackPlayerMsgReply]("player"))
+```
+
+### EVM Support
+
+Messages can be submitted by EVM smart contracts by using the `WithMsgEVMSupport` option when you register your messages. This will generate the ABI types necessary for interactions with smart contracts.
+
+```go
+import (
+    "pkg.world.dev/world-engine/cardinal"
+    "pkg.world.dev/world-engine/cardinal/message"
+)
+
+cardinal.RegisterMessage(w, "attack-player", message.WithMsgEVMSupport[msg.AttackPlayerMsg, msg.AttackPlayerMsgReply]())}
+```
```

Comment:
if we wanna be 100% correct, we'd need to import whatever `msg` is and i think this is the case with `query` below as well

maybe like "github.com/your-org/your-cardinal-project/message"? not too sure what would be best

Reply:
yeah that sounds fine

Reply:
can you make an issue to address the other code blocks?